proto/coprocessor_v2.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package coprocessor_v2;
+
+import "errorpb.proto";
+import "kvrpcpb.proto";
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+option java_package = "org.tikv.kvproto";
+
+
+message RawCoprocessorRequest {
+    kvrpcpb.Context context = 1;
+
+    string copr_name = 2;
+    string copr_version_constraint = 3;
+
+    bytes data = 4;
+}
+
+message RawCoprocessorResponse {
+    bytes data = 1;
+
+    errorpb.Error region_error = 2;
+    string other_error = 3;
```

Comment:
It would be better to add a comment for this error. If the coprocessor is not found or the version is not compatible, the error will be returned in this field.

Reply:
I think this kind of comment could be outdated very easily.

Reply:
If there are some known error types, I think it would be better to specify where it is in the protocol.

Reply:
Good point! Will add some known error messages as well.

Reply:
Also, a bit unrelated: @andylokandy @skyzh should we use 
```
message Error {
  oneof error {
    errorpb.Error region_error = 1;
    ....
    string other_error = n;
  }
}

message RawCoprocessorResponse {
  oneof message {
    bytes data = 1;
    Error error = 2;
  }
}
```
to indicate that there can either be a successful result or an error?